internal/core/context.rs

```diff
@@ -12,7 +12,7 @@ use alloc::rc::Rc;
 
 thread_local! {
     pub(crate) static GLOBAL_CONTEXT : once_cell::unsync::OnceCell<SlintContext>
-        = once_cell::unsync::OnceCell::new()
+        = const { once_cell::unsync::OnceCell::new() }
```

Comment:
The CI correctly point out that this doesn't work with the variant we use for no_std.
So you probably can change the thread_local! macro for unsafe_single_thread to include this const production.